Gradebook/Program.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+
+namespace Gradebook
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Please enter 5 student names.");
```

Comment:
You didn't really have to create the students and grades arrays, you could have put the students and grades directly in the dictionary.